src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.jsx

```diff
@@ -29,7 +29,7 @@ const Histogram = ({
 Histogram.propTypes = {
   data: PropTypes.array.isRequired,
   xAxisDataKey: PropTypes.string.isRequired,
-  barDataKey: PropTypes.array.isRequired,
```

Comment:
this was a bug

Reply:
Not sure if this is related to runtime error I'm seeing: 
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/113449836/208511113-bc57345c-31d0-4eb8-ac13-f41d4ee8dc4f.png">